hw/ip_templates/rv_core_ibex/lint/rv_core_ibex.vbl.tpl

```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# waiver file for ${module_instance_name}
+
+# Waive Ibex formal DV environment
+waive --rule=interface-name-style --location="mem.sv"
+waive --rule=module-port --location="spec_instance.sv"
```

Comment:
@marnovandermaas Verible flags errors in the current formal code. We're waiving this in OT as of this PR, because we mostly care about linting the RTL. Every waiver adds needs to be reviewed during sign-off, though, so it would be great if we didn't need this. Do you think we formal code can be adapted so that it's Verible-lint-clean?